config/crd/db/db.flanksource.com_redisdbs.yaml

```diff
@@ -1,5 +1,3 @@
-
----
```

Comment:
Can you get that the 'format' make target is being run?  CRDs that are not changed should not have diffs

Reply:
It is being run, with this changeset being the result of that